cli/src/pcluster/templates/cluster_stack.py

```diff
@@ -639,6 +640,7 @@ def _add_fsx_storage(self, id: str, shared_fsx: SharedFsx):
                 storage_type=shared_fsx.fsx_storage_type,
                 subnet_ids=self.config.compute_subnet_ids,
                 security_group_ids=self._get_compute_security_groups(),
+                tags=[CfnTag(key="Name", value=shared_fsx.name)],
```

Comment:
do we want to add a prefix for this like we do for other resources that we create as part of a cluster workflow?

Reply:
I prefer having the name tag without prefix because:
1. the customer would otherwise have two tags for the same concept: `Name` and `parallelcluster:Name`;
2. it makes sense to prefix non standard tags, but `Name` is a standard tag;

Reply:
Talked offline with @demartinofra about this. He also agrees on having 'Name' without prefix also because it would be displayed by default on the console.